packages/wonder-blocks-modal/docs.md

```diff
@@ -41,9 +60,9 @@ class ModalWrapper extends React.Component {
 </View>
 ```
 
-### Example: Modals with custom styles
+### Example: Dialogs with custom styles
```

Comment:
I think we should also have an example that shows us using the building blocks.  building-blocks.md would be a good place for this.

Reply:
Maybe build the two-column modal from https://www.figma.com/file/CRryTHqyr1l6aOOltD32KMh5/Modal-v2?node-id=132%3A0.

Reply:
Added to `building-blocks.md` as you suggested 😃